### ISSUE TITLE
Why is /graphql/ endpoint listed as an "admin" url?

### ISSUE THREAD

**Issue by kinglozzer** [tractorcow-farm/silverstripe-fluent#585]
https://github.com/tractorcow-farm/silverstripe-fluent/blob/4.4.4/src/Middleware/InitStateMiddleware.php#L29-L32

I’m not sure why `graphql/` is included here instead of `admin/graphql/`? It seems to prevent translations when querying data via graphql. I’ve had some success removing it with the following:

```yml
---
Name: app-fluent
---
TractorCow\Fluent\Middleware\InitStateMiddleware:
  admin_url_paths: null
---
After: '#app-fluent'
---
TractorCow\Fluent\Middleware\InitStateMiddleware:
  admin_url_paths:
    - 'dev/'

```

**Comment by robbieaverill** [comment]
My guess is that we wrote this before the route was changed to admin 🧐

**Comment by kinglozzer** [comment]
That was my suspicion... I’d be a bit anxious about changing this in a patch/minor release though - perhaps it’s better to add the above as a docs suggestion for now?

**Comment by tractorcow** [comment]
ah yeah, in ss 4.0 `/graphql` was for the CMS. In my apps I had to create second url for front-end graphql.

We can change this now I think, since it's `admin/graphql` and is safe. :)

**Comment by tractorcow** [comment]
Maybe we could add version detection to check the default url for admin graphql.

**Comment by tractorcow** [comment]
Reminder to self to fix this in 5.0 alpha 3
